Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -406,7 +412,10 @@ Neither will the `arbiter` placement be merged with the `all` placement to allow
 The remaining mons will still use the `mon` and/or `all` sections.
 
 !!! note
```

Comment:
Instead of a note, seems better to just make this a regular bullet-list, with an intro something like this:
```suggestion
Other considerations of the placement settings include:
```